workspace-manager-clienttests/src/main/java/scripts/disruptivescripts/DeleteInitialPods.java

```diff
@@ -0,0 +1,86 @@
+package scripts.disruptivescripts;
+
+import bio.terra.testrunner.common.utils.KubernetesClientUtils;
+import bio.terra.testrunner.runner.DisruptiveScript;
+import bio.terra.testrunner.runner.TestRunner;
+import bio.terra.testrunner.runner.config.TestUserSpecification;
+import io.kubernetes.client.openapi.ApiException;
+import io.kubernetes.client.openapi.models.V1Deployment;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.concurrent.TimeUnit;
+
+public class DeleteInitialPods extends DisruptiveScript {
+    private static final Logger logger = LoggerFactory.getLogger(DeleteInitialPods.class);
+
+    public DeleteInitialPods() {
+        super();
+        manipulatesKubernetes = true;
+    }
+
+    protected static final int secondsToWaitBeforeStartingDisrupt = 15;
+
+    @Override
+    public void setParameters(List<String> parameters) {
+
```

Comment:
this doesn't actually seem to set the parameters list.

Reply:
This is just a placeholder to show where parameters are set in case we need them for resiliency tests. For this example, no additional parameters is needed.